### DISCUSSION TITLE
Get all panels under a device

### DISCUSSION THREAD

**Discussion by BlindBadgerStudios** [MartinStoffel/tigo#3] [Ideas]
Just a thought, it would be nice to have all the panels under devices for each "inverter" (like one would see in the electrical view).  It would also keep the sensor list a bit cleaner for larger installs.  Otherwise, this was so easy and simple that it works great! (Even on the free plan right now, just less detail)

**Comment by MartinStoffel** [comment]
I'm not really a specialist on Tigo nor Home Assistant, but can this not be done simply by naming conventions? I only have a small setup with one inverter.
